main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -621,4 +629,68 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
         }
     }
   }
+
+  /**
+    * Package the current project and release it on GitHub.
+    */
+  def release(flix: Flix): Validation[Unit, BootstrapError] = {
+    val formatter = flix.getFormatter
+
+    // Ensure that we have a manifest
+    val manifest = optManifest match {
+      case Some(m) => m
+      case None => return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.MissingManifest))
+    }
+
+    // Check if `github` option is present
+    val githubRepo = manifest.repository match {
+      case Some(r) => r
+      case None =>
+        return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.NoLinkedProject))
+    }
+
+    // Check if `--github-key` option is present
+    val githubKey = flix.options.githubKey match {
+      case Some(k) => k
+      case None =>
+        return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.MissingApiKey))
+    }
+
+    if (!flix.options.assumeYes) {
+      // Ask for confirmation
+      print(s"Release ${formatter.blue(s"github:$githubRepo")} ${formatter.yellow(s"v${manifest.version}")}? [y/N]: ")
+      val response = readLine()
+      response.toLowerCase match {
+        case "y" => // Continue
+        case _ => return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.Cancelled))
```

Comment:
what if I accidentally write "yes"?
Should we require it to be one or the other?

Reply:
Worst case is you accidentally cancel and you can try again I guess

Reply:
If you write yes, you get to try again?

Reply:
I don't think there is a good way to deal with the description.

But note that you can just edit it on GitHub. 

This tooling takes part of the ugly bits with getting the files assembled and placed there.

Reply:
Should we just look at the first letter then? Or just specifically "yes"?

Reply:
Hell, lets allow "y" and "yes". Nothing else.